### DISCUSSION TITLE
UI Change

### DISCUSSION THREAD

**Discussion by jojobingsa** [YetiForceCompany/YetiForceCRM#16621] [How to]
Hey, 
where can I change the design/UI, I want to try to make a Moder UI for the system.

**Comment by mariuszkrzaczkowski** [comment]
layouts:
https://github.com/YetiForceCompany/YetiForceCRM/tree/developer/layouts/basic
https://github.com/YetiForceCompany/YetiForceCRM/tree/developer/public_html/layouts/basic

skins:
https://github.com/YetiForceCompany/YetiForceCRM/tree/developer/public_html/layouts/basic/skins
